<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_PlatComponent</class>
 <widget class="QWidget" name="Dlg_PlatComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dlg_PlatComponent</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="16" column="2">
      <widget class="QCheckBox" name="chk_cardNumberPos">
       <property name="text">
        <string>V:号牌坐标</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="chk_faceSpecial">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>人脸特写</string>
       </property>
      </widget>
     </item>
     <item row="14" column="3">
      <widget class="QCheckBox" name="chk_limitValue">
       <property name="text">
        <string>M:限速值</string>
       </property>
      </widget>
     </item>
     <item row="15" column="3">
      <widget class="QCheckBox" name="chk_picSum">
       <property name="text">
        <string>R:图片总数</string>
       </property>
      </widget>
     </item>
     <item row="14" column="4">
      <widget class="QCheckBox" name="chk_illegalType">
       <property name="text">
        <string>N:违章类型</string>
       </property>
      </widget>
     </item>
     <item row="14" column="5">
      <widget class="QCheckBox" name="chk_roadCode">
       <property name="text">
        <string>O:道路编码</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QCheckBox" name="chk_illegalCode">
       <property name="text">
        <string>L:违章代码</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_26">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1677215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_27">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_28">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1677</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>过车存储路径：</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_29">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>命名规则：</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_30">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>自定义：</string>
       </property>
      </widget>
     </item>
     <item row="15" column="5">
      <widget class="QCheckBox" name="chk_numberType">
       <property name="text">
        <string>T:号牌种类</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="chk_enable">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QCheckBox" name="chk_picType">
       <property name="text">
        <string>Q:图片类型</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_24">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>UUID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_25">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1677215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>上报数据</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QCheckBox" name="chk_direction">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>P:行驶方向</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="chk_kakPic">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>卡口图片</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QCheckBox" name="chk_cardNumber">
       <property name="text">
        <string>B:车牌号</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="chk_time">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>A:时间</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QCheckBox" name="chk_cardLogo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>F:车标</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4">
      <widget class="QCheckBox" name="chk_cardSpeed">
       <property name="text">
        <string>D:车速</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QCheckBox" name="chk_NumberColor">
       <property name="text">
        <string>C:车牌颜色</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="ledt_port_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="cmb_type_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: rgb(255,255,255);
    color: rgb(0, 0, 0);
border: 1px solid #333333;
}

</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QCheckBox" name="chk_laneNumber">
       <property name="text">
        <string>G:车道号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ledt_ip_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="chk_kakMergePic">
       <property name="text">
        <string>卡口合成图片</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QCheckBox" name="chk_position">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>K:地点</string>
       </property>
      </widget>
     </item>
     <item row="13" column="5">
      <widget class="QCheckBox" name="chk_cardType">
       <property name="text">
        <string>J:车辆类型</string>
       </property>
      </widget>
     </item>
     <item row="13" column="4">
      <widget class="QCheckBox" name="chk_checkCode">
       <property name="text">
        <string>I:防伪码</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="chk_carSpecial">
       <property name="text">
        <string>车辆特写</string>
       </property>
      </widget>
     </item>
     <item row="13" column="3">
      <widget class="QCheckBox" name="chk_deviceNumber">
       <property name="text">
        <string>H:设备编号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="chk_illegalPic">
       <property name="text">
        <string>违法图片</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QCheckBox" name="chk_illegalMeragePic">
       <property name="text">
        <string>违法合成图片</string>
       </property>
      </widget>
     </item>
     <item row="12" column="5">
      <widget class="QCheckBox" name="chk_cardColor">
       <property name="text">
        <string>E:车身颜色</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="chk_carInfo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>仅过车信息</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QCheckBox" name="chk_illegalVideo">
       <property name="text">
        <string>违法录像</string>
       </property>
      </widget>
     </item>
     <item row="15" column="4">
      <widget class="QCheckBox" name="chk_curPicNum">
       <property name="text">
        <string>S:当前图片数</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QCheckBox" name="chk_directionCode">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>U:方向代码</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="chk_numberSpecial">
       <property name="text">
        <string>号牌特写</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>违法存储路径：</string>
       </property>
      </widget>
     </item>
     <item row="8" column="6">
      <widget class="QCheckBox" name="chk_utf8">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>是否UTF8编码</string>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <widget class="QComboBox" name="cmb_spit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
    background-color: rgb(255,255,255);
    color: rgb(0, 0, 0);
border: 1px solid #333333;
}

</string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>分隔符：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="btn_expand_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>收缩</string>
       </property>
      </widget>
     </item>
     <item row="10" column="5">
      <widget class="QPushButton" name="btn_clear">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
     <item row="11" column="4">
      <widget class="QPushButton" name="btn_add_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="chk_snap">
       <property name="text">
        <string>场景空拍</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QCheckBox" name="chk_eventPic">
       <property name="text">
        <string>事件图片</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
